### ISSUE TITLE
CPU usage not calculated correctly

### ISSUE THREAD

**Issue by dwmarcuskim** [cts-2019f-nnstreamer/gst-shark#11]
### Error generating script:
```
$ GST_DEBUG="GST_TRACER:7" GST_TRACERS="live;cpuusage" \
gst-launch-1.0 \
    v4l2src name=cam_src ! videoconvert ! videoscale ! video/x-raw,width=640,height=480,format=RGB,framerate=30/1 ! tee name=t \
    t. ! queue leaky=2 max-size-buffers=2 ! videoscale ! video/x-raw,width=300,height=300,format=RGB ! tensor_converter ! \
        tensor_transform mode=arithmetic option=typecast:float32,add:-127.5,div:127.5 ! \
        tensor_filter framework=tensorflow-lite model=tflite_model/ssd_mobilenet_v2_coco.tflite ! \
        tensor_decoder mode=bounding_boxes option1=tflite-ssd option2=tflite_model/coco_labels_list.txt option3=tflite_model/box_priors.txt option4=640:480 option5=300:300 ! \
        compositor name=mix sink_0::zorder=2 sink_1::zorder=1 ! videoconvert ! ximagesink \
    t. ! queue leaky=2 max-size-buffers=10 ! mix.
````

### logs from gst-shark cpuusage tracer
```
0:00:04.270259476 20947 0x563329002b30 TRACE             GST_TRACER :0:: cpuusage, number=(uint)3, load=(double)87.755104;
0:00:05.269301778 20947 0x563329002b30 TRACE             GST_TRACER :0:: cpuusage, number=(uint)0, load=(double)84.210526;
0:00:05.269338984 20947 0x563329002b30 TRACE             GST_TRACER :0:: cpuusage, number=(uint)1, load=(double)85.416672;
0:00:05.269354040 20947 0x563329002b30 TRACE             GST_TRACER :0:: cpuusage, number=(uint)2, load=(double)82.474228;
0:00:05.269367397 20947 0x563329002b30 TRACE             GST_TRACER :0:: cpuusage, number=(uint)3, load=(double)85.714287;
```
### logs from LiveProfiler
```
CPU 0               34.7%
CPU 1               34.6%
CPU 2               34.5%
CPU 3               35.8%
```
get

**Comment by dwmarcuskim** [comment]
CPU Usage according to top is around 80%, seems LiveProfiler makes error currently.

**Comment by dwmarcuskim** [comment]
Fixed by PR #12
